srcs/layouts/latn_qwerty_haw.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Hawaiian)" script="latin">
+  <row>
+    <key key0="q" key2="1" key4="loc esc"/>
+    <key key0="w" key1="~" key2="2" key3="\@"/>
+    <key key0="e" key1="!" key2="3" key3="\#" key4="ē"/>
```

Comment:
I don't understand now we have `āēīū` on the layout but also the corresponding dead key. I'd prefer if only one solution is used.
I consider that swiping is faster than using a dead key. Dead keys are better if there are too many accented letters or if they are used less often. For Hawaiian I would suggest having `āēīū` as swipes.

Reply:
The key4 slot is empty anyway right? Is there a downside to leaving both options available so that the user can choose?
Normally phone keyboards have extra options on long press, and even if a layout has dead keys, accented letters are still available as long press options. This keyboard has swiping instead, so I added them as swipe instead of long press.

Reply:
The downsides of having too many options on layouts is that they get fuller and less readable. The keyboard also adds more keys to the layouts automatically depending on languages and options and that needs some empty space.

Reply:
There is still a lot of empty space in other places, like on the a s d f z x keys

Reply:
If you are fine with the layout, then let's merge.